CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -1,5 +1,18 @@
 # Changelog 1.13
 
+## 1.13.1 - 2023-08-24
+
+### Changed
+
+- Change the default flow's action to `drop` in ARPSpoofGuardTable to improve the efficiency in preventing ARP spoofing. ([#5378](https://github.com/antrea-io/antrea/pull/5378), [@hongliangl])
```

Comment:
"to improve..." -> "to effectively prevent ARP spoofing"

Reply:
Updated.